<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 234, 248);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <widget class="QTabWidget" name="InputImage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(201, 212, 235);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="InputScreen">
       <attribute name="title">
        <string>Input Screen</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="IPTCustomView" name="InputScreen_1">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(46, 52, 54);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="SelectStandardImage">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(143, 173, 147);</string>
              </property>
              <property name="text">
               <string>Select Standard Image</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QSlider" name="verticalSliderInputScreen1">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(232, 232, 168);
background-color: rgb(201, 183, 226);
selection-background-color: rgb(23, 45, 110);</string>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="1">
             <widget class="QPushButton" name="SelectTestImage">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(143, 173, 147);</string>
              </property>
              <property name="text">
               <string>Select Test Image</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="IPTCustomView" name="InputScreen_2">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(46, 52, 54);</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QSlider" name="verticalSliderInputScreen2">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(232, 232, 168);
background-color: rgb(201, 183, 226);
selection-background-color: rgb(23, 45, 110);</string>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="OutputScreen">
       <attribute name="title">
        <string>Output Screen</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(109, 133, 135);</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Inspected Image&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QFrame" name="frame">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(235, 222, 228);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QSlider" name="verticalSliderOutPutScreen">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(232, 232, 168);
background-color: rgb(201, 183, 226);
selection-background-color: rgb(23, 45, 110);</string>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="IPTCustomView" name="OutPutScreen">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>10</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(211, 215, 207);</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="3" column="0">
             <widget class="QListView" name="listView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777</width>
                <height>16777</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(238, 238, 236);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEdit">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(176, 219, 197);</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(109, 133, 135);</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Status of Image&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(109, 133, 135);</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Defect Types In Image&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_Defect_Image"/>
    <addaction name="separator"/>
    <addaction name="actionInputImageFitWindow"/>
    <addaction name="separator"/>
    <addaction name="actionOutputImageFitWindow"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_For_Update"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Defect_Image"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionInputImageFitWindow"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionOutputImageFitWindow"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Select Standard Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCheck_For_Update">
   <property name="text">
    <string>Check For Update</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionSelect_Defect_Image">
   <property name="text">
    <string>Select Test Image</string>
   </property>
  </action>
  <action name="actionInputImageFitWindow">
   <property name="text">
    <string>Input Image Fit Window</string>
   </property>
  </action>
  <action name="actionOutputImageFitWindow">
   <property name="text">
    <string>Output Image Fit Window</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IPTCustomView</class>
   <extends>QGraphicsView</extends>
   <header location="global">IPTCustomView.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
